Packet++/header/HttpLayer.h

```diff
@@ -457,14 +467,14 @@ namespace pcpp
 		}
 
 	private:
+		friend class HttpResponseFirstLine;
```

Comment:
Why do we need to add `HttpResponseFirstLine` as a friend class?

Reply:
sorry, I forgot to cleanup.